RHDEVDOCS 5960 Errata RN update for GitOps 1.9.4, 1.10.2, 1.11.1 - 1.11 and gitops-docs (#73653)
openshift/openshift-docs: modules/gitops-release-notes-1-11-1.adoc

{gitops-title} 1.11.1 is now available on {OCP} 4.12, 4.13, and 4.14.

[id="errata-updates-1-11-1_{context}"]
== Errata updates

[id="rhsa-2024-0689-gitops-1-11-1-security-update-advisory_{context}"]
=== RHSA-2024-0689 - {gitops-title} 1.11.1 security update advisory

Issued: 2024-02-05

The list of security fixes that are included in this release is documented in the following advisory:

* link:https://access.redhat.com/errata/RHSA-2024:0689[RHSA-2024-0689]

> If you have installed the {gitops-title} Operator, run the following command to view the container images in this release:

Reviewer: ```suggestion
If you have installed the {gitops-title} Operator, view the container images in this release by running the following command:
```

[source,terminal]
----
$ oc describe deployment gitops-operator-controller-manager -n openshift-operators
----

[id="fixed-issues-1-11-1_{context}"]
== Fixed issues

The following issue has been resolved in the current release:

> * Before this update, all versions of Argo CD `v2.9.2` and later were vulnerable to cross-server request forgery (CSRF) attacks. As a result, Argo CD would accept non-GET requests even if they did not specify their content type. This update fixes the issue by upgrading the Argo CD version to `v.2.9.5` and patching this vulnerability in the Argo CD API. link:https://issues.redhat.com/browse/GITOPS-4219[GITOPS-4219]

Reviewer: ```suggestion
* Before this update, all versions of Argo CD `v2.9.2` and later were vulnerable to cross-server request forgery (CSRF) attacks. As a result, Argo CD accepted non-GET requests even if they did not specify a content type. This update fixes the issue by upgrading the Argo CD version to `v.2.9.5` and patches this vulnerability in the Argo CD API. link:https://issues.redhat.com/browse/GITOPS-4219[GITOPS-4219]
```
